src/librustdoc/html/sources.rs

```diff
@@ -213,11 +212,10 @@ impl SourceCollector<'_, '_> {
                 print_src(
                     buf,
                     contents,
-                    cx.shared.edition(),
                     file_span,
                     cx,
                     &root_path,
-                    None,
+                    highlight::DecorationInfo::default(),
```

Comment:
Even though it could be simplified into `Default::default()`, I think it's personally much better this way. :)